analyzers/src/SonarAnalyzer.CFG/ShimLayer/IPropertySymbolExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2024 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace StyleCop.Analyzers.Lightup;
+
+public static class IPropertySymbolExtensions
```

Comment:
See also IFieldSymbolExtensions and others
https://github.com/SonarSource/sonar-dotnet/blob/0876a571b93c5748ea8396b9f29ba53a6e54543f/analyzers/src/SonarAnalyzer.CFG/ShimLayer/IFieldSymbolExtensions.cs#L7-L20